Casbin/Util/BuiltInFunctions.cs

```diff
@@ -15,6 +15,28 @@ public static class BuiltInFunctions
         private static readonly Regex s_keyMatch4Regex = new(@"\{([^/]+)\}");
         private delegate bool GFunction(string subject1, string subject2, string domain = null);
 
+        /// <summary>
+        /// KeyGet returns the matched part
+        /// For example, "/foo/bar/foo" matches "/foo/*"
+        /// "bar/foo" will been returned
+        /// </summary>
+        /// <param name="key1">The first argument.</param>
+        /// <param name="key2">The second argument.</param>
+        /// <returns>Whether key1 matches key2.</returns>
+        public static string KeyGet(string key1, string key2)
+        {
+            int m = key1.Length, n = key2.Length;
```

Comment:
`m` and `n` may not be the most suitable name, It is difficult to see the association with key1 or key2.